qubes/vm/qubesvm.py

```diff
@@ -1627,6 +1627,27 @@ def is_memory_balancing_possible(self):
                 return False
         return True
 
+    @property
+    def use_memory_hotplug(self):
+        """Use memory hotplug for memory balancing.
+        This is preferred if supported, because it has less initial overhead.
+        The support needs to be enabled in the VM's kernel.
+        """
+        feature = self.features.check_with_template('memory-hotplug', None)
+        if feature is not None:
+            return bool(feature)
+        # if not explicitly set, check if support is advertised
+        # for dom0-provided kernel - check there
+        if self.kernel is not None:
+            return os.path.exists(os.path.join(
+                self.storage.kernels_dir, 'memory-hotplug-supported'))
```

Comment:
```suggestion
            try:
                os.stat(os.path.join(
                    self.storage.kernels_dir, 'memory-hotplug-supported'))
                return True
            except FileNotFoundError:
                pass
```

Other errors should not be swallowed.